src/components/landing_page/LandingPage.tsx

```diff
@@ -22,6 +19,13 @@ const LandingPageView = ({ setView, view }: any) => {
     },
   }));
 
+  const onInitHome = function (typewriter: TypewriterClass) {
```

Comment:
You can use the `useRef` hook to store a reference to the Typewriter component and call its start method inside the `useEffect` hook instead of specifying the text and delay in the onInit prop. This can improve the performance of the component because the Typewriter component will not have to re-create the text on every render.